apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/createLogicApp/initLogicAppCodeProjectVScode/InitCodeProject.ts

```diff
@@ -0,0 +1,370 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import {
+  func,
+  projectLanguageSetting,
+  funcVersionSetting,
+  deploySubpathSetting,
+  tasksVersion,
+  tasksFileName,
+  launchVersion,
+  settingsFileName,
+  preDeployTaskSetting,
+  launchFileName,
+  extensionsFileName,
+  extensionCommand,
+  functionsExtensionId,
+} from '../../../../../../constants';
+import { ext } from '../../../../../../extensionVariables';
+import { localize } from '../../../../../../localize';
+import { isSubpath, confirmEditJsonFile, confirmOverwriteFile } from '../../../../../utils/fs';
+import {
+  getDebugConfigs,
+  getLaunchVersion,
+  isDebugConfigEqual,
+  updateDebugConfigs,
+  updateLaunchVersion,
+} from '../../../../../utils/vsCodeConfig/launch';
+import { updateWorkspaceSetting } from '../../../../../utils/vsCodeConfig/settings';
+import {
+  getTasksVersion,
+  updateTasksVersion,
+  updateTasks,
+  getTasks,
+  updateInputs,
+  getInputs,
+} from '../../../../../utils/vsCodeConfig/tasks';
+import { isMultiRootWorkspace } from '../../../../../utils/workspace';
+import { AzureWizardExecuteStep, nonNullProp } from '@microsoft/vscode-azext-utils';
+import type { IActionContext } from '@microsoft/vscode-azext-utils';
+import type {
+  ISettingToAdd,
+  IProjectWizardContext,
+  ProjectLanguage,
+  ITask,
+  ITaskInputs,
+  ITasksJson,
+  ILaunchJson,
+  IExtensionsJson,
+} from '@microsoft/vscode-extension';
+import { WorkflowProjectType, FuncVersion } from '@microsoft/vscode-extension';
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import type { TaskDefinition, DebugConfiguration, WorkspaceFolder } from 'vscode';
+
+export abstract class InitCodeProject extends AzureWizardExecuteStep<IProjectWizardContext> {
+  public priority = 20;
+  protected preDeployTask?: string;
+  protected settings: ISettingToAdd[] = [];
+
+  protected abstract executeCore(context: IProjectWizardContext): Promise<void>;
+  protected abstract getTasks(): TaskDefinition[];
+  protected getTaskInputs?(): ITaskInputs[];
+  protected getWorkspaceSettings?(): ISettingToAdd[];
+
+  protected getDebugConfiguration(version: FuncVersion): DebugConfiguration {
+    return {
+      name: localize('attachToNetFunc', 'Attach to Logic App'),
```

Comment:
Per [Microsoft Style Guide - Capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization): 

```suggestion
      name: localize('attachToNetFunc', 'Attach to logic app'),
```